crates/amaru/src/stages/mod.rs

```diff
@@ -80,14 +81,14 @@ pub type BlockHash = Hash<32>;
 #[derive(Clone)]
 pub enum StorePath<S> {
```

Comment:
Wonder if it would not make sense to rename this? `StorePath` implies the existence of a `Path`, `StoreType` might be better?

Reply:
I agree. I changed it to `StoreType`.